bogoworld/main.py

```diff
@@ -0,0 +1,58 @@
+from random import randint
+import sys, time
+
+#prints a string by:
+# printing a random character
+# checking against the current desired character
+# backspacing if the result was incorrect
+#
+#Arguments:
+# s - the string to be printed (ASCII-only)
+# d - the average delay between correctly printed characters (in seconds)
+def bogoprint(s, delay=0.2):
+
+ #generator comprehension that returns the numeric value of every character in a string, one at a time
+ g = (ord(char) for char in s)
+
+ try:
+  #pre-generate the first character
+  i = next(g)
+  while 1:
```

Comment:
While not incorrect, this is not very pythonic
```suggestion
  while True:
```